charts/argo-events/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 0.1.0
-appVersion: 1.6.3
+version: 0.2.0
+appVersion: 1.7.3
```

Comment:
We want to move to a model where appVersion == version. Do you you mind if we move to that model as part of that change?

Reply:
Sure! Updated.